There is some conflict of `spark-sql-kafka-0-10_2.12` and get errors when compiling.

```
# mvn clean package -DskipTests=true -Dscalatest.skip=true -Dwagon.skip=true -Dmaven.test.skip=true -pl external/kafka-0-10-sql -am
./dev/make-distribution.sh --name openmldbspark --pip --tgz -Phadoop-2.7 -Pyarn -Pallinone

[ERROR] [Error] /Users/tobe/code/4pd/spark/external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDelegationTokenSuite.scala:59:
method reset in class UserGroupInformation cannot be accessed in object org.apache.hadoop.security.UserGroupInformation
[ERROR] [Error] /Users/tobe/code/4pd/spark/external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala:382: method res
et in class UserGroupInformation cannot be accessed in object org.apache.hadoop.security.UserGroupInformation
[ERROR] two errors found
```

--- a/sql/core/pom.xml
+++ b/sql/core/pom.xml
@@ -168,6 +168,12 @@
       <groupId>com.4paradigm.hybridse</groupId>
       <artifactId>hybridse-native</artifactId>
       <version>${hybridse.native.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>com.4paradigm.openmldb</groupId>
@@ -175,11 +181,7 @@
       <version>${openmldb.batch.version}</version>
       <exclusions>
         <exclusion>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.apache.spark</groupId>
+          <groupId>*</groupId>
           <artifactId>*</artifactId>
         </exclusion>
       </exclusions>
@@ -272,21 +274,21 @@
         <activeByDefault>true</activeByDefault>
       </activation>
       <properties>
-        <openmldb.batch.version>${default.openmldb.batch.version}</openmldb.batch.version>
+        <openmldb.batch.version>${default.openmldb.batch.version}-SNAPSHOT</openmldb.batch.version>
         <hybridse.native.version>${default.hybridse.native.version}</hybridse.native.version>
       </properties>
     </profile>
     <profile>
       <id>macos</id>
       <properties>
-        <openmldb.batch.version>${default.openmldb.batch.version}-macos</openmldb.batch.version>
+        <openmldb.batch.version>${default.openmldb.batch.version}-macos-SNAPSHOT</openmldb.batch.version>
         <hybridse.native.version>${default.hybridse.native.version}-macos</hybridse.native.version>
       </properties>
     </profile>
     <profile>
       <id>allinone</id>
       <properties>
-        <openmldb.batch.version>${default.openmldb.batch.version}-allinone</openmldb.batch.version>
+        <openmldb.batch.version>${default.openmldb.batch.version}-allinone-SNAPSHOT</openmldb.batch.version>
         <hybridse.native.version>${default.hybridse.native.version}-allinone</hybridse.native.version>
       </properties>
     </profile>

